taskoutput/util.go

```diff
@@ -0,0 +1,36 @@
+package taskoutput
+
+import (
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/pail"
+	"github.com/evergreen-ci/utility"
+	"github.com/pkg/errors"
+)
+
+func newBucket(bucketName, bucketType string) (pail.Bucket, error) {
+	var b pail.Bucket
+	var err error
+
+	switch bucketType {
+	case evergreen.BucketTypeS3:
+		b, err = pail.NewS3Bucket(pail.S3Options{
+			Name:        bucketName,
+			Region:      "us-east-1",
```

Comment:
Why do we want this hardcoded? 
If we do, we should probably use a constant for "us-east-1".

Reply:
Good catch! Thanks.

Reply:
We only use `us-east-1` for our buckets so we can hard code this.